Feature/registration forms

luisabellan opened the pull request:
**ATTENTION**: For it to work correctly the following PR has to be accepted first: https://github.com/Lambda-School-Labs/LabsPT12-Express_Groomer-TeamA-BE/pull/16

- Created customer registration form 

- Created groomer registration form

coryortega commented on .env.sample:
```diff
@@ -0,0 +1,3 @@
+REACT_APP_CLIENT_ID=0oalwp37fU2aV9UEG4x6
+REACT_APP_OKTA_ISSUER_URI=https://auth.lambdalabs.dev/oauth2/default
+REACT_APP_API_URI=https://labspt12-express-groomer-a-api.herokuapp.com/
```
Need to remove .env.sample file

luisabellan replied:
I can't remove it, apparently the tests requires a .env.sample to be there:
https://gyazo.com/53f3c8e490df89e3304b5439de236328
https://gyazo.com/f0dc2fe8bd39703aecc29d44ae5a1c73

coryortega commented on src/components/Forms/Register.js:
```diff
@@ -0,0 +1,16 @@
+  }
+}
+
+export default Register;
```
Not entirely necessary, but could be converted to functional component

luisabellan replied:
done :+1:

coryortega commented on src/components/Layouts/Navbar.js:
```diff
@@ -0,0 +1,22 @@
+  }
+}
+
+export default Navbar;
```
Navbar might not be necessary, now that William implemented an Ant design dashboard with a nav on the side

luisabellan replied:
removed

coryortega commented on src/components/Forms/RegisterCustomer.js:
```diff
@@ -0,0 +1,306 @@
+  //     })
+  //     .catch(function (error) {
+  //         console.log(error)
+  //     })
```
While I think the comments are nice so Adella can see the changes made, we ultimately should delete them before merging to main

luisabellan replied:
done

coryortega reviewed (changes requested):
Nice job, just need to remove .env.sample file and also delete comments, refactor class component to functional component

luisabellan committed:
Update .gitignore

luisabellan committed:
Delete .env.sample

luisabellan committed:
converted Register class-based component to a functional component

luisabellan committed:
reorganized code

luisabellan committed:
save

luisabellan committed:
added some style to the registration forms

luisabellan committed:
Deleted Navbar

luisabellan committed:
Fixed markup

luisabellan committed:
Update .gitignore

luisabellan committed:
.env.sample require to pass tests?

luisabellan committed:
.env.sample required to pass tests?

luisabellan commented:
> Nice job, just need to remove .env.sample file and also delete comments, refactor class component to functional component

There must be something wrong on the tests, because it requires that there is a .env.sample file:
https://gyazo.com/f0dc2fe8bd39703aecc29d44ae5a1c73

luisabellan committed:
Update .env.sample

luisabellan committed:
Update .env.sample

luisabellan reviewed (commented):
.env.sample must be present so it can be copied into .env to recreate it (cp .env.sample .env)

luisabellan committed:
Update index.js

luisabellan committed:
Added links to homepage

Added links to customer registration, grromers registration, and registration (Are you a groomer or a customer)

luisabellan committed:
Update index.js

luisabellan committed:
done

* set registration back to false

* corrected route for NotFoundPage component

luisabellan committed:
corrected route

luisabellan committed:
routes fixed

luisabellan committed:
groomers dashboard  and customers dashboard routes fixed

* groomers dashboard  and customers dashboard routes
* also deleted links for which there are no components, like /datavis, and /profile-list, and /register

luisabellan committed:
Update index.js

coryortega committed:
Delete .env.sample

deleted .env.sample

luisabellan committed:
conflicts resolved

luisabellan committed:
Merge branch 'fix/routes' into main

luisabellan committed:
conflicts resolved
